doc/classes/Area.xml

```diff
@@ -122,8 +122,16 @@
 				Emitted when one of another Area's [Shape]s enters one of this Area's [Shape]s. Requires [member monitoring] to be set to [code]true[/code].
 				[code]area_rid[/code] the [RID] of the other Area's [CollisionObject] used by the [PhysicsServer].
 				[code]area[/code] the other Area.
-				[code]area_shape_index[/code] the index of the [Shape] of the other Area used by the [PhysicsServer]. Get the [CollisionShape] node with [code]area.shape_owner_get_owner(area_shape_index)[/code].
-				[code]local_shape_index[/code] the index of the [Shape] of this Area used by the [PhysicsServer]. Get the [CollisionShape] node with [code]self.shape_owner_get_owner(local_shape_index)[/code].
+				[code]area_shape_index[/code] the index of the [Shape] from the other Area.
+				[code]local_shape_index[/code] the index of the [Shape] from this Area.
+				[b]Example of getting the[/b] [CollisionShape] [b]node from the shape index:[/b]
```

Comment:
These **[b]** make me **mad** but my **fury shall be** directed toward**s whoe**ver wro**te thes**e on `master`, **g**o**o**d**n**e**s**s **m**e.